src/app-view.js

```diff
@@ -56,7 +68,14 @@ class AppView extends React.Component {
                         sections={this.state.sections}
                     />
                 </Sidebar>
-                <Content>
+                <Content
+                    style={{
+                        transform: `${
+                            this.state.isSidebarHidden ? 'translateX(0)' : 'translateX(-200px)'
```

Comment:
can we move this style change into css and change the classnames on state change?